### ISSUE TITLE
Run command block times out

### ISSUE THREAD

**Issue by LowPowerLab** [davidferguson/pibakery-raspbian#20]
Awesome tool, wish I had this long ago!

I am trying to use **Run command** for doing some apt-get update and installs. One of the Run Commands times out and besides not really having a visual indication of what step is failing, the real issue I see is the UI getting stuck and not being able to continue to flow if one of the blocks fails.

**Comment by GuinnessStoutly** [comment]
I agree with LowPowerLab.  Some of my recipes uninstall nearly every application and then do an update and upgrade during which time the RunCommand is at a certain percentage for a long time. Would be nice to be able to see what's going on or the ability to jump out to a cli window to see the processes running.

**Comment by kraigh** [comment]
I'm seeing the same issue- `apt-get update` especially is causing a timeout almost every time, as well as installing any packages. Any ideas how to fix this?

**Comment by jordan314** [comment]
Yeah my "connect to WIFI" block often times out. I'm not sure if it's hanging or if I'm seeing the screen while it finishes booting up. A "cancel" button would be handy.
